_posts/2015-09-15-foreman-community-newsletter-september.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Foreman Community Newsletter - September 2015
+date: '2015-09-15T13:01:00.000+03:00'
+author: Tomas Strachota
+tags:
+- katello
+- remote_execution
+- docker
+- openscap
+- foreman
+- dhcp
+- deep-dive
+- discovery
+- chef
+- demo
+- newsletter
+modified_time: '2015-10-06T13:43:53.260+03:00'
+blogger_id: tag:blogger.com,1999:blog-6789674575954398874.post-1715269517448905299
+blogger_orig_url: http://blog.theforeman.org/2015/09/foreman-community-newsletter-september.html
+---
+
+### Foreman 1.9 and 1.9.1 released
```

Comment:
Ah thanks.  Yes, I agree it will and it ultimately depends on the post content.

On another site I maintain I swap between both as sometimes if you're putting images or something into the text then you need to set a manual excerpt.